packages/run-protocol/src/psm/psm.js

```diff
@@ -0,0 +1,187 @@
+// @ts-check
+import '@agoric/zoe/exported.js';
+import '@agoric/zoe/src/contracts/exported.js';
+import '@agoric/governance/src/exported';
+import {
+  assertProposalShape,
+  ceilMultiplyBy,
+  floorDivideBy,
+  floorMultiplyBy,
+  makeRatio,
+} from '@agoric/zoe/src/contractSupport/index.js';
+import { Far } from '@endo/marshal';
+// import { CONTRACT_ELECTORATE } from '@agoric/governance';
+
+import { AmountMath } from '@agoric/ertp';
+import { makeMakeCollectFeesInvitation } from '../collectFees.js';
+
+const { details: X } = assert;
+
+/**
+ * @file The Parity Stability Module supports efficiently minting/burning a
+ * stable token at a specified fixed ratio to a reference stable token, which
+ * thereby acts as an anchor to provide additional stability. For flexible
+ * economic policies, the fee percentage for trading into and out of the stable
+ * token are specified separately.
+ *
+ */
+
+const BASIS_POINTS = 10000n;
+
+/**
+ * Stage a transfer of a singe asset from one seat to another, with an optional
+ * remapping of the Keywords. Check that the remapping is for the same amount.
+ *
+ * @param {ZCFSeat} from
+ * @param {ZCFSeat} to
+ * @param {AmountKeywordRecord} txFrom
+ * @param {AmountKeywordRecord} txTo
+ */
+function stageTransfer(from, to, txFrom, txTo = txFrom) {
+  assert(AmountMath.isEqual(Object.values(txFrom)[0], Object.values(txTo)[0]));
+  from.decrementBy(txFrom);
+  to.incrementBy(txTo);
+}
+
+/**
+ * @param {ZCF<{
+ *    anchorBrand: Brand,
+ *    anchorPerStable: Ratio,
+ *    main: {
+ *      WantStableFeeBP: bigint,
+ *      GiveStableFeeBP: bigint,
+ *      MintLimit: Amount } }>
+ * } zcf
+ * @param {{feeMintAccess: FeeMintAccess}} privateArgs
+ */
+export const start = async (zcf, privateArgs) => {
+  const { anchorBrand, anchorPerStable } = zcf.getTerms();
```

Comment:
Isn't `anchorBrand` term redundant with `anchorPerStable`? 
```suggestion
  const { anchorPerStable } = zcf.getTerms();
  const anchorBrand = anchorPerStable.numerator.brand;
```

Reply:
The `anchorBrand` should be in `anchorPerStable`, but one is a ratio and the other a brand.

Reply:
Added an assert instead.